contracts/0.8.25/vaults/VaultFactory.sol

```diff
@@ -119,6 +119,7 @@ contract VaultFactory {
 
         if (_roleAssignments.length > 0) dashboard.grantRoles(_roleAssignments);
 
+        dashboard.setNodeOperatorFeeRecipient(_nodeOperatorManager);
```

Comment:
let's provide a fee recipient for the dashboard initializer, assigning `_nodeOperatorManager`